src/lib/apiTypesV2/Attribute.cpp

```diff
@@ -52,7 +52,8 @@ std::string Attribute::render(ConnectionInfo* ciP, RequestType requestType, bool
     else
     {
       out = "{";
-      out += pcontextAttribute->toJson(true, renderMode, requestType);  // param 1 'true' as it is the last and only element
+      // FIXME PR: renderMode + attrsFormat ... a bit confusing. Should we join these two somehow? 
```

Comment:
Before unifiying them we should precisely now which values could take renderMode in order to see how to deail with the cartesian product bewteeen renderMode values and attrsFormat values. Makes sense?